CLITest.yml

```diff
@@ -0,0 +1,165 @@
+    - CLITest
+  always: true
+
+jobs:
+- job: LiveTestPython38
+  displayName: Live Test Python 3.8
+  timeoutInMinutes: 9999
+  strategy:
+    maxParallel: 20
+    matrix:
+      acr:
+        Target: acr
+      acs:
+        Target: acs
+      advisor:
+        Target: advisor
+      ams:
+        Target: ams
+      apim:
+        Target: apim
+      appconfig:
+        Target: appconfig
+      appservice:
+        Target: appservice
+      backup:
+        Target: backup
+      batch:
+        Target: batch
+      batchai:
+        Target: batchai
+      billing:
+        Target: billing
+      botservice:
+        Target: botservice
+      cdn:
+        Target: cdn
+      cloud:
+        Target: cloud
+      cognitiveservices:
+        Target: cognitiveservices
+      configure:
+        Target: configure
+      consumption:
+        Target: consumption
+      container:
+        Target: container
+      cosmosdb:
+        Target: cosmosdb
+      deploymentmanager:
+        Target: deploymentmanager
+      dla:
+        Target: dla
+      dls:
+        Target: dls
+      dms:
+        Target: dms
+      eventgrid:
+        Target: eventgrid
+      eventhubs:
+        Target: eventhubs
+      extension:
+        Target: extension
+      feedback:
+        Target: feedback
+      find:
+        Target: find
+      hdinsight:
+        Target: hdinsight
+      interactive:
+        Target: interactive
+      iot:
+        Target: iot
+      iotcentral:
+        Target: iotcentral
+      keyvault:
+        Target: keyvault
+      kusto:
+        Target: kusto
+      lab:
+        Target: lab
+      managedservices:
+        Target: managedservices
+      maps:
+        Target: maps
+      monitor:
+        Target: monitor
+      natgateway:
+        Target: natgateway
+      netappfiles:
+        Target: netappfiles
+      network:
+        Target: network
+      policyinsights:
+        Target: policyinsights
+      privatedns:
+        Target: privatedns
+      profile:
+        Target: profile
+      rdbms:
+        Target: rdbms
+      redis:
+        Target: redis
+      relay:
+        Target: relay
+      reservations:
+        Target: reservations
+      resource:
+        Target: resource
+      role:
+        Target: role
+      search:
+        Target: search
+      security:
+        Target: security
+      servicebus:
+        Target: servicebus
+      servicefabric:
+        Target: servicefabric
+      signalr:
+        Target: signalr
+      sql:
+        Target: sql
+      sqlvm:
+        Target: sqlvm
+      storage:
+        Target: storage
+      vm:
+        Target: vm
+  pool:
+    vmImage: 'Ubuntu-16.04'
+  steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.8'
+        addToPath: true
+        architecture: 'x64'
+    - bash: |
+        cd ..
+        pwd
+        ls
+        git clone https://github.com/Azure/azure-cli.git
+        git clone https://github.com/Azure/azure-cli-extensions.git
```

Comment:
can we run extension tests in a seperate pipeline?

Reply:
OK